typehints/microbit/microbit/__init__.pyi

```diff
@@ -160,12 +175,22 @@ class MicroBitTouchPin(MicroBitAnalogDigitalPin):
 pin0: MicroBitTouchPin
 """Pad 0."""
 
+pin0: MicroBitAnalogDigitalPin
```

Comment:
It looks like `pin0`, `pin1`, `pin2` are defined twice which is forbidden in Python stubs. `MicroBitTouchPin` already inherits from `MicroBitAnalogDigitalPin` so the type should be correct.